Mathlib/CategoryTheory/Limits/Shapes/ConcreteCategory.lean

```diff
@@ -0,0 +1,100 @@
+/-
+Copyright (c) 2023 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.CategoryTheory.Limits.Preserves.Shapes.Products
+import Mathlib.CategoryTheory.Limits.Shapes.Types
+
+/-!
+# Limits in concrete categories
+
+In this file, we combine the description of limits in `Types` and the API about
+the preservation of products and pullbacks in order to describe these limits in a
+concrete category `C`.
+
+If `F : J → C` is a family of objects in `C`, we define a bijection
+`Limits.Concrete.productEquiv F : (forget C).obj (∏ F) ≃ ∀ j, F j`.
```

Comment:
I believe that this fulfills a `TODO` in `Limits/ConcreteCategory.lean`. Also, some results from that file might have a better home in this new file.

Reply:
Thanks for the suggestion! I have moved the results about specific shapes (multiequalizers, etc.) into this new file.